mev-programs/scripts/autosnapshot_inner.sh

```diff
@@ -269,8 +275,8 @@ main() {
   check_env_vars_set
 
   # make sure snapshot location exists and has a genesis file in it
-  mkdir -p "$SNAPSHOT_DIR"
-  cp "$LEDGER_DIR"/genesis.bin "$SNAPSHOT_DIR"
+  mkdir -p "$SNAPSHOT_OUTPUT_DIR"
```

Comment:
probably should also create the archive dir

Reply:
that gets created by rpc

Reply:
i can add check in systemd to see if it exits